client/src/components/fhirpipe/index.tsx

```diff
@@ -0,0 +1,255 @@
+  const sources = dataSources ? dataSources.sources : [];
+  const resources = dataResources ? dataResources.resources : [];
+
+  const SourceSelect = Select.ofType<Source>();
+  const ResourceMultiSelect = MultiSelect.ofType<Resource>();
+
+  const isItemSelected = (list: any[], item: any): boolean => {
+    return list.includes(item);
+  };
+
+  const handleSourceSelect = (source: Source): void => {
+    setSelectedSource(source);
+  };
+  const handleResourceSelect = (resource: Resource): void => {
+    if (!isItemSelected(selectedResources, resource)) {
+      setSelectedResources([...selectedResources, resource]);
+    } else {
+      selectedResources.splice(selectedResources.indexOf(resource), 1);
+      setSelectedResources([...selectedResources]);
+    }
+  };
+
+  const renderSource = (source: Source, { handleClick }: any) => (
+    <MenuItem
+      key={source.id}
+      text={source.name}
+      label={source.template.name}
+      onClick={handleClick}
+      shouldDismissPopover={false}
+    />
+  );
+
+  const renderResource = (resource: Resource, { handleClick }: any) => (
+    <MenuItem
+      key={resource.id}
+      text={resource.definition.type}
+      label={resource.label}
+      icon={isItemSelected(selectedResources, resource) ? 'tick' : 'blank'}
+      onClick={handleClick}
+      shouldDismissPopover={false}
+    />
+  );
+
+  const handleTagRemove = (_value: string, index: number): void => {
+    selectedResources.splice(index, 1);
+    setSelectedResources([...selectedResources]);
+  };
+
+  const filterSource: ItemPredicate<Source> = (query, source) => {
+    return source.name.toLowerCase().indexOf(query.toLowerCase()) >= 0;
+  };
+
+  const onFormSubmit = async (
+    e: React.FormEvent<HTMLFormElement>
+  ): Promise<void> => {
+    e.preventDefault();
+
+    // Get the credentials
+    const {
+      data: {
+        source: { credential }
+      }
+    } = await client.query({
+      query: qCredentialForSource,
+      variables: { sourceId: selectedSource.id }
+    });
+
+    if (!credential) {
+      toaster.show({
+        message: 'You need provide credentials for the source database.',
+        intent: 'danger',
+        icon: 'warning-sign',
+        timeout: 4000
+      });
+      return;
+    }
+
+    // The possible params for running fhirpipe are:
+    // mapping, sources, resources, labels, reset_store,
+    // chunksize, bypass_validation, multiprocessing
+    // It is also needed to provide credentialId
+    const body = {
+      sources: selectedSource.name,
+      resources: selectedResources.map(r => r.definition.type),
+      labels: selectedResources.map(r => r.label),
+      credentialId: credential.id,
+      bypass_validation: bypassValidation
+    };
+    const headers = { 'Content-Type': 'application/json' };
+    console.log(body);
+    axios
+      .post(`${FHIRPIPE_URI}/run`, body, { headers: headers })
+      .then((res: any) => {
+        console.log(`statusCode: ${res.statusCode}`);
```

Comment:
more feedback would be nice, but we can leave that for an other PR (how many rows were processed, warnings etc...)